### PR TITLE
Flag to see all NewReport events

### PR DESCRIPTION
**Pull request by tallywiesenberg**
Some data types are very important, but rarely reported. This change adds a flag that alerts on all NewReport events on the given chain.

Implements click, unimplements argparse, and re-implements the `--wait` option as a `click` option

### CODE REVIEW

**Review by oraclown — CHANGES_REQUESTED** [review]
Thanks @tallywiesenberg ! Left a refactoring suggestion. Also why migration to click? It's a big import. Not against it, just curious about the reason

File: src/tellor_disputables/alerts.py [hunk 1]
```diff
@@ -1,16 +1,50 @@
...
+            else:
+                msg = generate_alert_msg(False, new_report.link)
+        else:
+            msg = generate_alert_msg(False, new_report.link)
+
+        send_text_msg(twilio_client, recipients, from_number, msg)
+
+    else:
+        # Account for unsupported queryIDs
+        if new_report.disputable is not None:
```
[the remark this answers is not in the document]
Comment (tallywiesenberg) [reply src/tellor_disputables/alerts.py:33]: true! corrected

**Comment by tallywiesenberg** [comment]
> Thanks @tallywiesenberg ! Left a refactoring suggestion. Also why migration to click? It's a big import. Not against it, just curious about the reason

Ah, I used click to be consistent with `telliot-feeds` and because I like the decorator functionality.

**Review by oraclown — APPROVED** [review]
Thanks @tallywiesenberg

**Pull request merged**